diff --git a/mobile/dev-ngrok.sh b/mobile/dev-ngrok.sh
index 6dd163ae32aa58ebe99919dbdefafe08b69283f4..14cac00241cd0c088b3ea0976be3ab1043d08acd 100755
--- a/mobile/dev-ngrok.sh
+++ b/mobile/dev-ngrok.sh
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 set -e
 
-echo "Starting ngrok tunnel on port 8002..."
-ngrok http 8002
+echo "Starting ngrok tunnel on port 8000..."
+ngrok http 8000
  
\ No newline at end of file
